src/ConstructiveSolidGeometry/IO.jl

```diff
@@ -231,7 +232,7 @@ function get_origin(::Type{T}, dict::AbstractDict, unit::Unitful.Units) where {T
     haskeyssum = sum(haskeys)
     if haskeyssum == 1
         o = dict[origin_names[i]]
-        if o isa Vector
+        if o isa AbstractVector
```

Comment:
When a `CSG.Geometry` is converted to a `Dict` using the method `CSG.Dictionary`, then the `origin` is saved as `CartesianPoint`. If one tries to create a `CSG.geometry` from that dictionary it ignores the `origin`, as it is not a `Vector` but `<: AbstractVector`. The method to parse the `origin` vector from a `Dict` was generalized to avoid this issue.